packages/python/accern_data/accern_data.py

```diff
@@ -246,6 +247,7 @@ def __init__(
         self._chunk_size = chunk_size
         self._buffer: List[pd.DataFrame] = []
         self._buffer_size = 0
+        csv.field_size_limit(sys.maxsize)
```

Comment:
sys.maxsize can be arbitrarily large and cause other problems. Maybe we can declare a constant and set it to a fairly large value (10x the number that is causing this issue) and set the limit to this constant.